docs/sync_code.md

```diff
@@ -0,0 +1,34 @@
+# Basic workflow for Operator
+
+1. Create a branch:
+Start by creating a branch from "main" branch.
+
+2. Ensure local tests pass:
+Run unit and e2e tests, and ensure they pass before submitting a pull request (PR) targeting "main" branch.
+If the PR is associated with any Jira ticket, include the ticket link in the "Description" section of the PR.
+
+
+3. Review and merge PR:
+Once the PR has passed github checks and is reviewed and approved, the PR author should merge it into "main" branch.
+
+4. Create a downstream sync PR:
+The PR author should then create another PR targeting "rhoai" branch. Preferably, the PR title should include the prefix [sync], and the "rhoai" label should be added to help reviewers quickly acknowledge it.
+If the PR is associated with any Jira ticket, include the ticket link in the "Description" section of the PR.
+
+5. Merge sync PR:
+After the sync PR has passed github checks and is reviewed and approved, the PR author should merge it into "rhoai" branch.
+
+6. Automated sync:
```

Comment:
not sure it should be in the ODH repo. Does not affect any related workflow according to the description.

Reply:
put it here only as information.
if everyone is clear about this, because i got asked many times in the past, i can remove it